package.json

```diff
@@ -1418,14 +1433,26 @@
 					"command": "issue.copyGithubPermalink",
 					"when": "github:hasGitHubRemotes"
 				},
+				{
+					"command": "issue.copyGithubPermalinkFile",
```

Comment:
Now we will have two commands that show with the same title in the command palette. Is there ever a case where we don't want the file + line included now? There's a proliferation of copy commands and it would be nice to remove all `*File` ones if the differentiation isn't needed.

Reply:
>  Is there ever a case where we don't want the file + line included now? 

My understanding is we don't want the file and line included if the command is invoked from the `file/share` menu--at least, that's the behavior we have for copying vscode.dev links. I don't know if this is extensively used though and I did get feedback that this behavior is unexpected.